spring/src/test/java/org/axonframework/spring/eventhandling/tokenstore/jpa/JpaTokenStoreTest.java

```diff
@@ -16,17 +16,17 @@
 
 package org.axonframework.spring.eventhandling.tokenstore.jpa;
 
-import org.axonframework.common.jpa.EntityManagerProvider;
-import org.axonframework.common.jpa.SimpleEntityManagerProvider;
+import org.axonframework.common.legacyjpa.EntityManagerProvider;
+import org.axonframework.common.legacyjpa.SimpleEntityManagerProvider;
 import org.axonframework.common.transaction.Transaction;
 import org.axonframework.common.transaction.TransactionManager;
-import org.axonframework.eventhandling.tokenstore.jpa.JpaTokenStore;
-import org.axonframework.eventhandling.tokenstore.jpa.TokenEntry;
+import org.axonframework.eventhandling.tokenstore.legacyjpa.JpaTokenStore;
+import org.axonframework.eventhandling.tokenstore.legacyjpa.TokenEntry;
 import org.axonframework.serialization.TestSerializer;
 import org.hibernate.dialect.HSQLDialect;
 import org.hibernate.jpa.HibernatePersistenceProvider;
-import org.junit.jupiter.api.*;
-import org.junit.jupiter.api.extension.*;
+import org.junit.jupiter.api.Test;
```

Comment:
I assume there's no equivalent of this in the `hibernate-6-integrationtests` module, as that module doesn't include Spring, right?

Reply:
Added on in `axon-spring-boot-3-integration-tests`.